src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -351,7 +351,7 @@ static function ( $event ) use ( $date_object ) {
 
 			$day_data = [
 				'date'             => $day_date,
-				'is_start_of_week' => $start_of_week === $date_object->format( 'w' ),
+				'is_start_of_week' => $start_of_week === (int)$date_object->format( 'w' ),
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Expected 1 space(s) after cast statement; 0 found
* :warning: Line exceeds 80 characters; contains 90 characters

<sub>hash: 5951af89d8c80a397cbec4047a036736</sub>